### ISSUE TITLE
Combine your networking ideas

### ISSUE THREAD

**Issue by nztdeadpool** [HanlinMo/Rotation-Invariant-Coordinate-Convolutional-Neural-Network#6]
I'd like to incorporate the network ideas you've devised, especially the rotation invariant coordinate system, into other models, but I'm running into state dictionary mismatches, does this mean I must train the network from scratch?

**Comment by HanlinMo** [comment]
Hello, I think that you can load pretrain parameters in our RIC-CNN models, because the baseline CNN and the corresponding RI-CNN have the same number of learnable parameters. But you need to check the layers' and weights' names in CNN and RI-CNN, because they are different. Also, we did not use bias parameters for RIC-Conv.

**Comment by nztdeadpool** [comment]
> Hello, I think that you can load pretrain parameters in our RIC-CNN models, because the baseline CNN and the corresponding RI-CNN have the same number of learnable parameters. But you need to check the layers' and weights' names in CNN and RI-CNN, because they are different. Also, we did not use bias parameters for RIC-Conv.

Thanks so much！！Regarding the essence of your design - the deformation convolution part, the coordinate system generation you designed is a new part of the design in the convolution layer, I should add their weights to be applicable in the new model, right? Because I'm also experiencing a lack of weights in the naming of the coordinate system generation.

**Comment by HanlinMo** [comment]
Yes, you need add the coordinate system's weights (they are constant and do not need to learn).

**Comment by nztdeadpool** [comment]
> Yes, you need add the coordinate system's weights (they are constant and do not need to learn).

Special thanks to you！I'll keep going
